proposals/nnnn-structured-concurrency.md

```diff
@@ -196,11 +242,11 @@ One of the variables for a given `async let` must be awaited at least once along
 {
   async let result = try fetchHTTPContent(of: url)
   if condition {
-    let header = await result.header
+    let header = await try result.header
     // okay, awaited `result`
   } else {
     // error: did not await 'result' along this path. Fix this with, e.g.,
-    //   _ = await result
+    //   _ = await try result
```

Comment:
I believe these were missing?

A few lines above we say:

> If the initializer of the `async let` can throw an error, **then each reference to a variable declared within that `async let` is considered to throw an error**, and therefore must also be enclosed in one of `try`/`try!`/`try?`. 

So we declared the result as `async let ... = try` so any reference to it must be as if throwing, is that right?

Or am I misreading this?

--- 

Is the wording 

> then each reference to a variable declared within that `async let` is considered to throw an error

correct or is it

> then each reference to a variable declared **by** that `async let` is considered to throw an error

?

Reply:
"by" is definitely the right one here, thank you